SliderControl.js

```diff
@@ -135,6 +138,9 @@ L.Control.SliderControl = L.Control.extend({
                 }
             }
         });
+        if (!_options.range && _options.alwaysShowDate) {
+            $('#slider-timestamp').html(_options.markers[24].feature.properties.time.substr(0, 19));
```

Comment:
I'm not sure why "24" works. But it also works with others smaller lists of points.